onboarding.md

```diff
@@ -0,0 +1,360 @@
+
+`nvm` is a version manager for node. It's not strictly required, but it will make it easier to switch from one node version to another.
+
+
+##### Linux
+
+Ensure that you have `curl` or `wget` installed, and run:
+
+```bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash 
+```
+or with wget:
+```bash
+wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
+```
+
+Then follow the rest of the MacOS installation steps.
+
+##### macOS
+
+Use a `brew` package manager
+
+```bash
+brew install nvm
+```
+and follow instructions shown in terminal to properly setup your shell environment for `nvm`.
+
+Or download and install the nvm installer with curl:
+
+```bash
+curl https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash
+```
+
+If the install was successful, run `nvm --version` which should return the version of nvm installed.
+
+Install and use the latest LTS version of NodeJS: _(at the time of writing this is `10.15.3`)_
+```bash
+nvm install --lts
+nvm use --lts
+```
+
+##### Windows
+[todo]
+
+##### Set Up NVM In `~/.bash_profile`, `.zshrc`, or `.profile`:
+
+For NVM to work the next time you open your shell, you need to make sure that following exists in your `.bash_profile`, `.zshrc`, or `.profile`:
+
+```bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" # This loads nvm
+```
+
+If you don't have a `~/.bash_profile`, you can create one with:
+```bash
+touch ~/.bash_profile
+```
+And using a your text editor of choice, add the following:
+```bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" # This loads nvm
+```
+
+You will need to run `source ~/.bash_profile`, or restart your shell for these changes to take effect.
+
+
+#### <a name='NPM'></a>NPM
+
+NPM is the node package manager. It allows us to install and manage the dependencies for each of the sub-projects.
+We also use npm as the entry point for many different commands, such as running tests [finish]
+
+If you installed `nvm` above, you can skip this step, as `npm` will already be installed.
+
+
+##### macOS
+
+Use homebrew to install `node` and `npm` together.
+
+```bash
+brew install node
+```
+
+If you have already installed `nvm` (see above) you can skip this step. Elsewhere you can have multiple Node.js instances and it could bring some problems in future.
+
+##### Linux
+
+```bash
+#Add the LTS Repo
+sudo apt-get install curl python-software-properties
+curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
+
+sudo apt-get install nodejs
+
+```
+
+For other flavours of Linx, follow the installation guide [here](https://nodejs.org/en/download/package-manager/)
+
+
+##### Windows
+
+[todo]
+
+
+
+## <a name='Non-CLITools'></a>Non-CLI Tools
+
+1. [Github](#1-github)
+2. [Postman](#2-postman)
+3. [Zenhub](#3-zenhub)
+4. [Javascript IDE](#4-javascript-ide) (optional)
+5. [MySQLWorkbench](#5-mysqlworkbench) (optional)
```

Comment:
proposing TTK as we're not actively supporting postman (tests) moving forward

Reply:
Let's get rid of this file, It's covered in the docs already and is out of date